sklearn/inspection/_ice.py

```diff
@@ -0,0 +1,491 @@
+        second call::
+
+          >>> from sklearn.inspection import (
+          ...                   plot_individual_conditional_expectation)
+          >>> from sklearn.datasets import make_friedman1
+          >>> from sklearn.linear_model import LinearRegression
+          >>> X, y = make_friedman1()
+          >>> est = LinearRegression().fit(X, y)
+          >>> disp1 = plot_individual_conditional_expectation(est,
+          ...                                 X)  # doctest: +SKIP
+          >>> disp2 = plot_individual_conditional_expectation(est, X,
+          ...                                 ax=disp1.axes_)  # doctest: +SKIP
+
+    Read more in the :ref:`User Guide <individual_conditional_expectation>`.
+
+    Parameters
+    ----------
+    estimator : BaseEstimator
+        A fitted estimator object implementing :term:`predict`,
+        :term:`predict_proba`, or :term:`decision_function`.
+        Multioutput-multiclass classifiers are not supported.
+
+    X : {array-like or dataframe} of shape (n_samples, n_features)
+        The data to use to build the grid of values on which the individual
+        conditional expectation will be evaluated.
+        This is usually the training data.
+
+    features : list of {int, str}
+        The target features for which to create the ICEs.
+        if any entry is a string, then it must be in ``feature_names``.
+
+    feature_names : array-like of shape (n_features,), dtype=str, default=None
+        Name of each feature; feature_names[i] holds the name of the feature
+        with index i.
+        By default, the name of the feature corresponds to their numerical
+        index for NumPy array and their column name for pandas dataframe.
+
+    target : int, optional (default=None)
+        - In a multiclass setting, specifies the class for which the ICEs
+          should be computed. Note that for binary classification, the
+          positive class (index 1) is always used.
+        - In a multioutput setting, specifies the task for which the ICEs
+          should be computed.
+
+        Ignored in binary classification or classical regression settings.
+
+    response_method : 'auto', 'predict_proba' or 'decision_function', \
+            optional (default='auto')
+        Specifies whether to use :term:`predict_proba` or
+        :term:`decision_function` as the target response. For regressors
+        this parameter is ignored and the response is always the output of
+        :term:`predict`. By default, :term:`predict_proba` is tried first
+        and we revert to :term:`decision_function` if it doesn't exist.
+
+    n_cols : int, optional (default=3)
+        The maximum number of columns in the grid plot. Only active when `ax`
+        is a single axis or `None`.
+
+    grid_resolution : int, optional (default=100)
+        The number of equally spaced points on the axes of the plots, for each
+        target feature.
+
+    centre : bool, optional (default=True)
```

Comment:
this parameter is in a different order relative to percentiles in `individual_conditional_expectation`

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16164/commits/a4e676208745edfd7058524dcab8d49d72598674